src/subscription/subscribe.js

```diff
@@ -106,16 +110,33 @@ function subscribeImpl(
   fieldResolver,
   subscribeFieldResolver
 ) {
-  const subscription = createSourceEventStream(
+  // If arguments are missing or incorrect, throw an early error.
+  assertValidExecutionArguments(
```

Comment:
I'll add more comments here - the idea is to throw an early error so you get an immediate error with stack since bad arguments would be programmer error, not a graphql query issue - and you want to know that ASAP, not waiting until you `.next().then()`